### DISCUSSION TITLE
I hope marine organisms can generate random sizes, which is more interesting!

### DISCUSSION THREAD

**Discussion by Hlwyefeng** [SirSquidly/Oceanic-Expanse#74] [Ideas]
The various organisms in the ocean are fixed in size, why not let them generate different sizes? I think this can also enhance the richness of the ocean!
Related mod references
https://www.minecraftforum.net/forums/mapping-and-modding-java-edition/minecraft-mods/wip-mods/1443050-wip-1-7-10forge-fantastic-fish-alpha-v1-4
![fish](https://github.com/SirSquidly/Oceanic-Expanse/assets/114284270/47c6fc43-d42b-45a5-8519-568d591dcd8a)
![fish](https://github.com/SirSquidly/Oceanic-Expanse/assets/114284270/8038235d-8199-471d-88bd-64f3005f02d8)

**Comment by SirSquidly** [comment]
A few mobs already have size variance included (such as Salmon, Tropical Slimes), and while I will be adding a few others with some variance, I recommend looking into dedicated mobs for randomized scaling if you want many or all mobs to have it
